src/common/scoped_message_writer.h

```diff
@@ -91,7 +91,7 @@ class scoped_message_writer
     {
       m_flush = false;
 
-      MCLOG(m_log_level, "msgwriter", m_oss.str());
+      MCLOG_FILE(m_log_level, "msgwriter", m_oss.str());
 
       if (epee::console_color_default == m_color)
```

Comment:
Poor wording. I was suggesting that `scoped_message_writer` use the standard logging procedure with the `std::cout` calls removed. I may have missed a reason for writing directly to `std::cout` - it just seems that this is not following standard practices within the codebase.